### PR TITLE
Batch export for model predictions

### PR DESCRIPTION
**Pull request by ardila**
* Add support for batch exporting a model run

### CODE REVIEW

File: tests/test_prediction.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def test_repr(test_object: any):
 def model_run(CLIENT):
     ds = CLIENT.create_dataset(TEST_DATASET_NAME)
     ds_items = []
-    for url in TEST_IMG_URLS:
+    for url in TEST_IMG_URLS[:2]:
```
Comment (sasha-scale) [inline tests/test_prediction.py:53]: remove slicing?

**Review by sasha-scale — APPROVED** [review]
LGTM barring nit

File: tests/test_prediction.py (hunk 1 above)
Comment (ardila) [reply tests/test_prediction.py:53]: It makes the tests faster and they all still pass :)

Comment (sasha-scale) [reply tests/test_prediction.py:53]: haha ok fine w/ me

**Pull request merged**